build.gradle

```diff
@@ -169,6 +170,7 @@ allprojects {
       googleJavaFormat()
       formatAnnotations()
     }
+  }
   }
 
```

Comment:
Alternately, we could handle this [the way I did in Randoop](https://github.com/randoop/randoop/pull/1072/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R280-R285).  Which is better, or is there an even better solution?

Reply:
It's better to disable it, because it's not defined what happens if no spotless configuration is specified.  (Or at least a quick skim for the documentation doesn't say what happends.

Why can't spotless be used with Java 8?

Reply:
> Why can't spotless be used with Java 8?

Spotless uses the latest version of google-java-format that is compatible with the currently running JVM.  Under Java 8, that is an older version that produces different formatting than more recent versions of google-java-format that run under Java 11+.